cli/asc.js

```diff
@@ -678,6 +678,19 @@ exports.main = function main(argv, options, callback) {
     const passes = [];
     function add(pass) { passes.push(pass); }
 
+    // enchance options for inline passes
+    if (optimizeLevel >= 2 && shrinkLevel === 0) {
+      module.setAlwaysInlineMaxSize(12);
+      module.setFlexibleInlineMaxSize(70);
+      module.setOneCallerInlineMaxSize(200);
+    } else {
+      shrinkLevel >= 2
+        ? module.setAlwaysInlineMaxSize(2)
+        : module.setAlwaysInlineMaxSize(4);  // default:  2
+      module.setFlexibleInlineMaxSize(65);   // default: 20
+      module.setOneCallerInlineMaxSize(80);  // default: 15
```

Comment:
Is this safe in a situation like `optimizeLevel=0, shrinkLevel=1`?

Reply:
In case with `-O0s` it will be
```js
setAlwaysInlineMaxSize(4);
setFlexibleInlineMaxSize(65);
setOneCallerInlineMaxSize(80);
```
Why this not safe?

Reply:
Wondering because `setAlwaysInlineMaxSize=2` by default, which is more conservative on size, and because I'm not sure about the size implications of raising `setFlexibleInlineMaxSize` :)

Reply:
I could set `setAlwaysInlineMaxSize(2)` for all shrink level `> 0`. Is it will be ok?

Reply:
Sounds conservative enough, yeah. Do you know how `setFlexibleInlineMaxSize` works? Can this increase size or is it more of a hint for how hard Binaryen is allowed to work?

Reply:
`setFlexibleInlineMaxSize` it seems use some smart heuristics and applied for general cases. Also it seems it bound `setOneCallerInlineMaxSize` but not sure.